cli/src/commands/schema/edit.ts

```diff
@@ -750,6 +750,9 @@ function parseDefaultValue(type: string, val: string): string | undefined {
   } else if (type === 'link') {
     return val ? String(val) : undefined;
   } else if (type === 'datetime') {
+    // Date fields have special values
+    if (['now'].includes(val)) return val;
```

Comment:
Adding a fast exit if one of the aliases are detected (like now which is not a date), leaving the door open to add more in the future.